### DISCUSSION TITLE
Local auth

### DISCUSSION THREAD

**Discussion by totoCZ** [netbirdio/netbird#753] [Issue Triage]
It looks like a good project (more stable than netmaker), but I'm confused about SSO.
Is it a dependency? Is there no local auth support?
Because SSO is something I don't plan to run.
(If it ran without docker then it'd be even more awesome.)

**Comment by totoCZ** [comment]
maybe https://www.authelia.com/ is better than keycloak but it still adds a lot of work to setup.

**Comment by hadleyrich** [comment]
It requies an identity provider (SSO) as a dependency so if you want to run it you will need to plan to run one.
